dnf/cli/option_parser.py

```diff
@@ -277,6 +277,8 @@ def _main_parser(self):
         main_parser.add_argument("-6", dest="ip_resolve", default=None,
                                  help=_("resolve to IPv6 addresses only"),
                                  action="store_const", const='ipv6')
+        main_parser.add_argument("--destdir", dest="destdir", default=None,
```

Comment:
@hoschiCZ I am really sorry, but I didn't noted that the option "--downloaddir" was in yum and in yum-utils. Please can you add also ``--downloaddir`` as alias to provide compatibility with YUM and also with "dnf download" command?
Like: main_parser.add_argument("--destdir", "--downloaddir", dest="destdir", default=None,